docs/contracts/overview/DigitalAssets.md

```diff
@@ -136,6 +136,114 @@ You can filter the `DataChanged` event with the following `dataKey`:
+
+bytes32 constant _TOKEN_ID_TO_SET = 0xcafecafecafecafecafecafecafecafecafecafecafecafecafecafecafecafe;
+
+function setMultipleDataForSingleTokenId(
+    ILSP8 lsp8Contract,
+    bytes memory lsp4MetadataValue,
+    bytes memory nftIconValue,
+    bytes memory nftMarketPlaceURLsValue
+) {
+    bytes32[] memory tokenIdsToUpdate = new bytes32[](3);
+    bytes32[] memory dataKeysToSet = new bytes32[](3);
+    bytes[] memory dataValuesToSet = new bytes[](3);
+
+    // we are setting 3 x data key-value pairs for the same tokenid
+    tokenIdsToUpdate[0] = _TOKEN_ID_TO_SET;
+    tokenIdsToUpdate[1] = _TOKEN_ID_TO_SET;
+    tokenIdsToUpdate[2] = _TOKEN_ID_TO_SET;
+
+    dataKeysToSet[0] = _LSP4_METADATA_KEY;
+    dataKeysToSet[1] = _NFT_ICON_DATA_KEY;
+    dataKeysToSet[2] = _NFT_MARKET_PLACE_URLS__DATA_KEY;
+
+    dataValuesToSet[0] = lsp4MetadataValue;
+    dataValuesToSet[1] = nftIconValue;
+    dataValuesToSet[2] = nftMarketPlaceURLsValue;
+
+    lsp8Contract.setDataBatchForTokenIds(
+        tokenIdsToUpdate,
+        dataKeysToSet,
+        dataValuesToSet
+    );
+}
+```
+
+**case 2: set the LSP4 Metadata of 3 x different tokenIds**
+
+```solidity
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.12;
+
+import {
+    ILSP8IdentifiableDigitalAsset as ILSP8
+} from "@lukso/lsp-smart-contracts/contracts/LSP8IdentifiableDigitalAsset/ILSP8IdentifiableDigitalAsset.sol";
+import {
+    _LSP4_METADATA_KEY
+} from "@lukso/lsp-smart-contracts/contracts/LSP4DigitalAssetMetadata/LSP4Constants.sol";
+
+bytes32 constant _FIRST_TOKEN_ID_TO_SET = 0xcafecafecafecafecafecafecafecafecafecafecafecafecafecafecafecafe;
+bytes32 constant _SECOND_TOKEN_ID_TO_SET = 0xbeefbeefbeefbeefbeefbeefbeefbeefbeefbeefbeefbeefbeefbeefbeefbeef;
+bytes32 constant _THIRD_TOKEN_ID_TO_SET = 0xf00df00df00df00df00df00df00df00df00df00df00df00df00df00df00df00d;
+
+function setMultipleDataForSingleTokenId(
+    ILSP8 lsp8Contract,
+    bytes memory firstTokenIdLsp4MetadataValue,
+    bytes memory secondTokenIdLsp4MetadataValue,
+    bytes memory thirdTokenIdLsp4MetadataValue
+) {
+    bytes32[] memory tokenIdsToUpdate = new bytes32[](3);
+    bytes32[] memory dataKeysToSet = new bytes32[](3);
+    bytes[] memory dataValuesToSet = new bytes[](3);
+
+    tokenIdsToUpdate[0] = _FIRST_TOKEN_ID_TO_SET;
+    tokenIdsToUpdate[1] = _SECOND_TOKEN_ID_TO_SET;
+    tokenIdsToUpdate[2] = _THIRD_TOKEN_ID_TO_SET;
+
+    // we are setting the metadata for 3 x different tokenIds
+    dataKeysToSet[0] = _LSP4_METADATA_KEY;
+    dataKeysToSet[1] = _LSP4_METADATA_KEY;
+    dataKeysToSet[2] = _LSP4_METADATA_KEY;
+
+    dataValuesToSet[0] = firstTokenIdLsp4MetadataValue;
+    dataValuesToSet[1] = secondTokenIdLsp4MetadataValue;
+    dataValuesToSet[2] = thirdTokenIdLsp4MetadataValue;
+
+    lsp8Contract.setDataBatchForTokenIds(
+        tokenIdsToUpdate,
+        dataKeysToSet,
+        dataValuesToSet
+    );
+}
+```
+
```

Comment:
```suggestion
  </TabItem>
</Tabs>
```

Reply:
There is already a `</TabItem>` tag below, with an other tab opened for ethers v6 after.